database/seed.js

```diff
@@ -59,11 +59,11 @@ const seedingActivities = () => {
         for (let j = 0; j < 12; j++) {
             let infoObj = {
                 rating: generateRandFloat(3.5, 5),
-                listing_type: faker.lorem.sentance,
+                activity_name: faker.lorem.sentence(),
```

Comment:
do you care about the word range? w/o a range it is going to be between 3-10 letters, something to think about later, but maybe not as important right now.

Reply:
also whoops, i must've looked at your first commit rather than the second one. my bad!

Reply:
no worries, good idea on the range, i will look into that thx